### DISCUSSION TITLE
Ability to change color of path in track and map

### DISCUSSION THREAD

**Discussion by faint069** [progweb/gpx2video#35] [Ideas]
Hello!
Great job. Thank you, for your app!
I think it would be useful to change color of path in track and map widgets from layout. 
Right now it's hardcoded to blue in track.cpp. And also this color is used both in map and track widget, there is no ability to set different colors for them.
I thin than there is all instruments to set up color from layout. So maybe it will be not so hard.
Thank you again..

**Comment by faint069** [comment]
Oh, forget another thing.
Ability to change units name, how it is rendered. 
Right now we can change units, but it will always render like km/h, for example. 
Maybe it is possible to change this from layout. 
Main reason is that I would like to use different alphabet to render overlay and I can change widget name, but units names are still in English. 
Of course right now I changed it in code and build version, but maybe it will be useful ti make maybe special section in layout file with unit names.

**Reply by progweb** [reply]
```
      <widget x="250" y="450" width="600" height="120" position="left" align="vertical">
          <type>speed</type>   
          <name>VITESSE</name> 
          <margin>20</margin>  
          <padding>5</padding> 
          <unit>kph</unit>     
      </widget>  
```

Unit can be mph, kph... and it should be well printed as it is rendered.
But indeed, unit label can't be edited.

```
src/videowidget.cpp:
std::string VideoWidget::unit2string(VideoWidget::Unit unit)
VideoWidget::Unit VideoWidget::string2unit(std::string &s)
```

At last, you can disable unit label display

```
          <unit>kph</unit>     
          <with-unit>false</with-unit>     
```

and use a simple text widget to add your own label.

I'll add a "unit-label" to override default label.

**Reply by faint069** [reply]
Oh, I really didn't think about text widget, that can work. 
But if there will be unit-label to override it will be great, thank you!

**Comment by progweb** [comment]
About track color, it's planned... In fact, I want to display a multi color track depended on the speed (or other telemetry data as elevation...)

**Reply by faint069** [reply]
That will be great! Thanks!
